jelares/portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,20 @@
 .title {
-    background-image: linear-gradient(to right, var(--gradColor1), var(--gradColor2));
-    height:30px;
-    padding-top:30px;
-    padding-bottom:30px;
-    padding-left:15px;
-    margin-bottom:50px;
+  background-image: linear-gradient(
+    to right,
+    var(--gradColor1),
+    var(--gradColor2)
+  );
+  height: 30px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 15px;
```

Comment:
I think there's a shorthand property for padding, which is recommended by CSS style

Reply:
This is also Jesus's file that I am planning on removing from the commit.

Reply:
Done